pymobiledevice3/cli/developer.py

```diff
@@ -773,6 +773,22 @@ def accessibility():
     """ accessibility options. """
     pass
 
+@accessibility.command('run-audit', cls=Command)
+@click.argument('test_type', required=False)
```

Comment:
use the `nargs=-1` for dealing with multiple arguments

Reply:
Made the changes for this